pysat/utils/io.py

```diff
@@ -0,0 +1,1156 @@
+                                        file_format=file_format,
+                                        epoch_name=epoch_name,
+                                        decode_timedelta=decode_timedelta,
+                                        labels=labels)
+
+    return data, meta
+
+
+def load_netcdf_pandas(fnames, strict_meta=False, file_format='NETCDF4',
+                       epoch_name='Epoch',
+                       labels={'units': ('units', str),
+                               'name': ('long_name', str),
+                               'notes': ('notes', str), 'desc': ('desc', str),
+                               'plot': ('plot_label', str),
+                               'axis': ('axis', str), 'scale': ('scale', str),
+                               'min_val': ('value_min', np.float64),
+                               'max_val': ('value_max', np.float64),
+                               'fill_val': ('fill', np.float64)}):
+    """Load netCDF-3/4 file produced by pysat in a pandas format.
+
+    Parameters
+    ----------
+    fnames : str or array_like
+        Filename(s) to load
+    strict_meta : bool
+        Flag that checks if metadata across fnames is the same if True
+        (default=False)
+    file_format : str
+        file_format keyword passed to netCDF4 routine.  Expects one of
+        'NETCDF3_CLASSIC', 'NETCDF3_64BIT', 'NETCDF4_CLASSIC', or 'NETCDF4'.
+        (default='NETCDF4')
+    epoch_name : str
+        Data key for time variable (default='Epoch')
+    labels : dict
+        Dict where keys are the label attribute names and the values are tuples
+        that have the label values and value types in that order.
+        (default={'units': ('units', str), 'name': ('long_name', str),
+        'notes': ('notes', str), 'desc': ('desc', str),
+        'min_val': ('value_min', np.float64),
+        'max_val': ('value_max', np.float64), 'fill_val': ('fill', np.float64)})
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        Class holding file data
+    meta : pysat.Meta
+        Class holding file meta data
+
+    Raises
+    ------
+    KeyError
+        If epoch/time dimension could not be identified.
+    ValueError
+        When attempting to load data with more than 2 dimensions or if
+        `strict_meta` is True and meta data changes across files.
+
+    See Also
+    --------
+    load_netcdf
+
+    """
+    # Ensure inputs are in the correct format
+    fnames = pysat.utils.listify(fnames)
+    file_format = file_format.upper()
+
+    # Initialize local variables
+    saved_meta = None
+    running_idx = 0
+    running_store = []
+    two_d_keys = []
+    two_d_dims = []
+    meta = pysat.Meta(labels=labels)
+
+    # Load data for each file
+    for fname in fnames:
+        with netCDF4.Dataset(fname, mode='r', format=file_format) as data:
+            # Build a dictionary with all global ncattrs and add those
+            # attributes to a pysat meta object
```

Comment:
```suggestion
            # Build a dictionary with all global ncattrs and add those
            # attributes to a pysat meta object. Any custom Meta attributes 
            # are automatically transferred to the holding pysat.Instrument 
            # object by pysat.
```
I went down a path here on Meta attributes and ended up verifying a design choice that I'd forgotten.